openshift/template.yaml

```diff
@@ -140,6 +140,9 @@ objects:
       - name: filter
         value: header.canonical('X-Secret-Token').compareSecret('X_SECRET_TOKEN',
           'cad-pd-token')
+      - kind: NamespacedInterceptor
+        name: cad-interceptor
+        ref: null
```

Comment:
Ew, this looks odd. I'm not sure this would work. I think the issue is the indentation of `ref` in the edited file above.